testing/ostest/posixtimer.c

```diff
@@ -170,8 +173,9 @@ void timer_test(void)
     }
 
 #ifndef SDCC
-  printf("timer_test: oact.sigaction=%p oact.sa_flags=%x oact.sa_mask=%jx\n",
-          oact.sa_sigaction, oact.sa_flags, (uintmax_t)oact.sa_mask);
+  printf("timer_test: oact.sigaction=%p oact.sa_flags=%x "
+         "oact.sa_mask=" SIGSET_FMT "\n",
+          oact.sa_sigaction, oact.sa_flags, SIGSET_ELEM(&oact.sa_mask));
```

Comment:
```suggestion
  printf("timer_test: oact.sigaction=%p oact.sa_flags=%x "
         "oact.sa_mask=" SIGSET_FMT "\n",
         oact.sa_sigaction, oact.sa_flags, SIGSET_ELEM(&oact.sa_mask));
```
BTW, very strange why it is hidden under `#ifndef SDCC`